db/models/product.js

```diff
@@ -1,33 +1,36 @@
 'use strict'
 
 const Sequelize = require('sequelize')
+const {STRING, TEXT, INTEGER} = require('sequelize')
 
 module.exports = db => db.define('product', {
```

Comment:
I think this should be products because all the models have the db.define part as plural